Linux.md

```diff
@@ -96,3 +97,9 @@
 
 ### <a name="moveCursorToTheEndOfLine"></a> Moves the cursor to the end of the line
 `Ctrl+e`
+
+### <a name="previousSuperuserCommand"></a> Runs Previous command used as superuser
+`$!!`
+
+### <a name="divideFileColumn"></a> Divide Files into Columns
```

Comment:
remove divideFileColumn command

Reply:
Removed the command check now @arverma007